tests/e2e/fixtures/statsFixtures.js

```diff
@@ -462,6 +519,49 @@ const seasonTwoGames = [
     p1: 'player2',
   },
 ];
+const seasonFourGames = [
+  // Week 1
+  {
+    name: 'Hearts Week 1, Game 1',
+    status: false,
+    p0Ready: true,
+    p1Ready: true,
+    passes: 0,
+    turn: 12,
+    isRanked: false,
+    result: 0,
+    updatedAt: dayjs().subtract(1, 'week').valueOf(),
+    p0: 'player1',
+    p1: 'player2',
+  },
+  // Week 2
+  {
+    name: 'Hearts Week 2, Game 1',
+    status: false,
+    p0Ready: true,
+    p1Ready: true,
+    passes: 0,
+    turn: 12,
+    isRanked: false,
+    result: 0,
+    updatedAt: dayjs().subtract(2, 'week').valueOf(),
```

Comment:
Just to make sure it is after the start of the current season let's add a day
```suggestion
    updatedAt: dayjs().subtract(2, 'week').add(1, 'day').valueOf(),
```